ecosystem/scenarios/test_requests.py

```
@@ -0,0 +1,212 @@
+        print(f"   ⚠️  Unexpected dependencies found: {unexpected_deps}")
+        # Don't fail on unexpected deps, just warn
+
+    print("   ✓ All required dependencies found")
+
+    # Run smoke tests by importing the bundled module
+    print("\n🧪 Running smoke tests with bundled library...")
+
+    try:
+        with load_bundled_module(bundled_output, "requests_bundled") as requests:
+            run_smoke_tests(requests)
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)